app/validators/sections/section_validator.py

```diff
@@ -116,23 +119,9 @@ def validate_question(self, block_or_variant):
 
             self.errors += question_validator.validate()
 
-            for answer in question.get("answers", []):
-                answer_validator = get_answer_validator(
-                    answer, self.questionnaire_schema
-                )
-
-                answer_validator.validate()
-
-                if question.get("summary") and answer["type"] not in [
-                    "TextField",
-                    "Checkbox",
-                    "Number",
-                ]:
-                    self.add_error(
-                        error_messages.UNSUPPORTED_QUESTION_SUMMARY_ANSWER_TYPE,
-                        answer_id=answer["id"],
-                    )
-                self.errors += answer_validator.errors
+            self._validate_answers(
+                question, QuestionnaireSchema.get_answers_from_question(question)
```

Comment:
Do this inline, call `QuestionnaireSchema.get_answers_from_question` inside _validate_answer` since you are already passing question.